web/src/main/ui/app/components/flows-new/edit-flow/edit-flow.component.ts

```diff
@@ -1,16 +1,18 @@
-import { Component, OnInit, OnDestroy, ViewChild } from '@angular/core';
```

Comment:
The ViewChild was added as part of the following: https://github.com/marklogic/marklogic-data-hub/pull/2689
I don't think it should be removed.